### DISCUSSION TITLE
怎么查看各个应用的硬盘空间占用呢？

### DISCUSSION THREAD

**Discussion by IshuaWang** [1Panel-dev/1Panel#7512] [Q&A]
我的服务器空间是60G，安装了一些应用，现在空间快满了，但是我感觉使用的存储量远没有这么多。有一些应用安装了之后又被卸载了，但是我也没有做备份，所以应该不是这个问题。目前可以看到每个应用占用了多少硬盘空间吗？

**Comment by wanghe-fit2cloud** [comment]
查看并清理一下 docker 镜像，如图：
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/1016507f-882b-4a9a-b604-33a96010dbfa" />
